src/openforms/prefill/service.py

```diff
@@ -97,8 +97,11 @@ def inject_prefill(
 
         if prefill_value != default_value and default_value is not None:
             logger.info(
-                "Overwriting non-null default value for component %r",
-                component,
+                "prefill.overwrite_non_null_default_value",
+                submission_uuid=str(submission.uuid),
+                component_type=component["type"],
+                default_value=default_value,
+                component_id=component.get("id"),
```

Comment:
Do we need the `.get()` here?Is there a possibility that there is no id configured?

Reply:
yeah there's a possibility, too many people manually mess with the JSON of the components :(